src/components/GoalScreenHeader/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import Images from '../../constants/images/Image';
+import { GoalScreenHeaderStyle } from './GoalScreenHeaderStyle';
+
+export type GoalScreenHeaderProps = {
```

Comment:
Again types in a separate file, if can Or even this is good enough as it's exclusive to this component. Your choice

Reply:
keeping it here only as it's exclusive to the component